tests/kernel/bigquery_tests.py

```diff
@@ -49,15 +49,43 @@ class TestCases(unittest.TestCase):
   @mock.patch('google.datalab.Context.default')
   def test_udf_cell(self, mock_default_context, mock_notebook_environment):
     env = {}
+    mock_default_context.return_value = TestCases._create_context()
+    mock_notebook_environment.return_value = env
+
+    with self.assertRaises(Exception):
```

Comment:
The raised exception is a generic `Exception` object though. How important do you think this is in unit tests anyway, where most of the underlying layers are mocked out, and only the one function's logic should be tested?